### PR TITLE
design doc

### PR DESCRIPTION
**Pull request by gingerwizard**

### CODE REVIEW

**Comment by gingerwizard** [comment]
This needs the blog link added in first paragraph before merge.

File: docs/DESIGN.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# Introduction
+
+This issue details the current design and implementation for ClickHouse-kafka-connect and how it aims to achieve exactly-once delivery semantics.
```
Comment (Ryado) [inline docs/DESIGN.md:3]: replace issue by document

File: docs/DESIGN.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+# Requirements
+
+For the reasons behind the development of this connector, see [here]().
```
[the remark this answers is not in the document]
Comment (gingerwizard) [reply docs/DESIGN.md:7]: yes see my first comment

[the remark this answers is not in the document]
Comment (mshustov) [reply docs/DESIGN.md:7]: That was a comment for myself 😄

[the remark this answers is not in the document]
Comment (gingerwizard) [reply docs/DESIGN.md]: done

**Review by tom-clickhouse — APPROVED** [review]

**Pull request merged**